src/test/java/nl/dtl/fairmetadata4j/utils/MetadataUtilsTest.java

```diff
@@ -140,6 +140,10 @@ public void testParseNoTitleStatement() throws Exception {
         metadata.setTitle(null);    
         MetadataUtils.getStatements(metadata);
         fail("This test is execpeted to throw an error");
+    }    
+    
+    public void testNoTitleWithFlag() {
+        
     }
```

Comment:
Just wondering: why did you commit an empty method? I see the next commit is the implementation, so why commit the empty method?